pkg/arvo/gen/hood/mount.hoon

```diff
@@ -7,11 +7,21 @@
 ::::
   ::
 :-  %say
+=>  |%
+    +$  bath
+      $@  desk
+      (lest knot)
+    --
 |=  $:  [now=@da eny=@uvJ bec=beak]
-        [[pax=path pot=$@(~ [v=@tas ~])] ~]
+        [=bath pot=$@(~ [v=@tas ~])]
+        ~
     ==
-?~  pot
-  =+  bem=(need (de-beam pax))
-  $(pot ~[?^(s.bem (rear s.bem) q.bem)])
-:-  %kiln-mount
-[pax v.pot]
+=/  bem=beam
+  ?@  bath  [bec(q bath) /]
```

Comment:
Shouldn't this expand to `[bec(q bath, r da+now) /]` to cover the "ran the generator while `|cd`'d into a not-latest revision case? I imagine we only want to mount non-live desks explicitly (ie `|mount %` in that case).

Reply:
I'm not entirely sure what you mean by that last sentence, but I think if run from a non-live checkout, you should still be able to `|mount %foo` and have that mean the latest revision of the `%foo` desk, which would imply that I should indeed make the change you're suggesting.

Reply:
We already support mounting older revisions of desks. I was just reiterating the proposed change by saying "we don't want to mount older revisions if the user doesn't explicitly say so in the command arguments".